src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -166,6 +166,414 @@ E. Click the *Tasks* tab, select an *Preview_Results* task and click on the *Pre
+
+```python
+#%connect(host=YOUR_HOST, port=YOUR_PORT, login=YOUR_LOGIN, password=YOUR_PASSWORD)
+```
+
+==== Using config file:
+
+For automatic sign in, create a file named *proactive_config.ini* in your notebook location.
+
+Fill your configuration file according to the format:
+
+```ini
+[proactive_server]
+host=YOUR_HOST
+port=YOUR_PORT
+[user]
+login=YOUR_LOGIN
+password=YOUR_PASSWORD
+```
+
+Save your file changes and restart the proactive kernel.
+
+You can also force the current Kernel to connect using any .ini config file through the `#%connect()` pragma:
+
+```python
+#%connect(path=PATH_TO/YOUR_CONFIG_FILE.ini)
+```
+
+(for more information about this format please check
+https://docs.python.org/3/library/configparser.html[configParser])
+
+=== Usage
+
+===== Creating a Python task
+
+To create a task, please use the pragma `#%task()` followed by the task implementation script wrote into a notebook block code.
+To use this pragma, at least, a task name has to be provided. Example:
+
+```python
+#%task(name=myTask)
+print('Hello world')
+```
+
+General usage:
+
+```python
+#%task(name=TASK_NAME, [language=SCRIPT_LANGUAGE], [dep=[TASK_NAME1,TASK_NAME2,...]], [generic_info=[(KEY1,VAL1), (KEY2,VALUE2),...]], [export=[VAR_NAME1,VAR_NAME2,...]], [import=[VAR_NAME1,VAR_NAME2,...]], [path=IMPLEMENTATION_FILE_PATH])\n'
+```
+
+As seen in the general usage, users can also provide more information about the task by using the pragma's options:
+
+====== Language
+
+The `language` parameter is needed when the task script is not written in native Python, the default language.
+The handled programming languages are:
+
+* Linux_Bash
+* Windows_Cmd
+* DockerCompose
+* Scalaw
+* Groovy
+* Javascript
+* Jython
+* Python
+* Ruby
+* Perl
+* PowerShell
+* R
+
+Example of usage when the task is written in `Linux_Bash`:
+
+```bash
+#%task(name=myTask, language=Linux_Bash)
+echo 'Hello, World!'
+```
+
+====== Dependencies
+
+One of the most important notions in workflows is the dependencies between tasks. To specify this information, please use the
+`dep` parameter. The value should be a list of all the tasks on which the new task depends. Example:
+
+```python
+#%task(name=myTask,dep=[parentTask1,parentTask2])
+print('Hello world')
+```
+
+====== Generic information
+
+To specify the values of the advanced ProActive variables
+https://doc.activeeon.com/latest/user/ProActiveUserGuide.html#_generic_information[generic_information],
+ the parameter `generic_info` is provided. The value should be a tuples `(key,value)` list of all the names and values of the
+ProActive parameters. Example:
+
+```python
+#%task(name=myTask, generic_info=[(var1,value1),(var2,value2)])
+print('Hello world')
+```
+
+====== Export/import variables
+
+The `export` and `import` parameters make possible variables propagation between the different tasks of a workflow.
+If `myTask1` variables `var1` and `var2` are needed in `myTask2`, the `myTask1` pragma should include and `export` with
```

Comment:
the `myTask1` pragma should include an (not and) `export` parameter (or option ?) ...
... 
and the `myTask2` pragma should include an `import` parameter (or option ?) with a list of the same names (of instead of including)